synapse_rock/rockcraft.yaml

```diff
@@ -13,6 +13,13 @@ license: Apache-2.0
 platforms:
   amd64:
 parts:
+    add-user:
```

Comment:
Rockcraft now injects an user to run services. You can see an example at https://github.com/canonical/discourse-k8s-operator/blob/main/discourse_rock/rockcraft.yaml#L9

Reply:
I do not think we can use that, as we need the user to have uid 991 and gid 991 and also to have a home (for gpg to work).